internal/handlers/common/projection.go

```diff
@@ -142,7 +161,7 @@ func projectDocument(inclusion bool, doc *types.Document, projection *types.Docu
 				return err
 			}
 
-		case float64, int32, int64: // field: number
+		case float64: // field: number
```

Comment:
Can we handle that case in our linter? I think if several types are in present in one `case`, they should be the in the right order, and the first type should be in the right place, but not second and following types

Reply:
linter has been submitted, now such cases are valid